tests/Cookie/Middleware/EncryptCookiesTest.php

```diff
@@ -30,12 +36,12 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        $container = new Container;
-        $container->singleton(EncrypterContract::class, function () {
+        $this->container = new Container;
+        $this->container->singleton(EncrypterContract::class, function () {
             return new Encrypter(str_repeat('a', 16));
         });
 
-        $this->router = new Router(new Dispatcher, $container);
+        $this->router = new Router(new Dispatcher, $this->container);
     }
```

Comment:
I notice this test lacks a `tearDown` method for unsetting the container and router properties from memory. Can you add that?